Client/SMTP_client.h

```diff
@@ -0,0 +1,176 @@
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <Windows.h>
+#include <WinSock2.h>
+#include <WS2tcpip.h>
+#include <iostream>
+#include <string>
+#include <openssl/ssl.h>
+#include <openssl/bio.h>
+#include <openssl/err.h>
+#include "base64.h"
+#include <ctime>
```

Comment:
ctime is a file of a standart library.
Order your includes in header file as follows: 
System files.
Other 3-rd party libraries' .h files.
Your project's .h files.